Add a guide for Migrating to v1 (#2518)
thoughtbot/administrate: docs/migrating-to-v1.md

---
title: Migrating to v1
---

In the release of v1.0.0, we [began bundling assets in the gem itself][2397].

> In future, this should mean that changes to the way you, or Administrate
> manages assets should not have an effect on you. However, this is quite a big

Reviewer: ```suggestion
In the future, this should mean that any changes made to assets (coming
from you or from Administrate updates) should not have an effect on you.
However, this is quite a big
```
